038-MdwCovid19/Coach/00-PreReqs.md

```diff
@@ -0,0 +1,131 @@
+| Azure SQL Database       | Microsoft.SQL                           |
+| Azure Storage            | Microsoft.Storage                       |
+| Azure Data Lake Store    | Microsoft.DataLakeStore                 |
+| Azure Virtual Machines   | Microsoft.Compute                       |
+| Azure Synapse            | Microsoft.Synapse                       |
+
+> Note:  Resource Provider Registration can be found at https://portal.azure.com/_yourtenantname_.onmicrosoft.com/resource/subscriptions/_yoursubscriptionid_/resourceproviders
+
+## Attendee Computers
+
+Attendees will be required to install software on the workstations that they are performing the WhatTheHack on. Ensure they have adequate permissions to perform software installation.
+
+## Deployment Instructions 
+
+1. Open a **PowerShell 7** window, run the following command, if prompted, click **Yes to All**:
+
+   ```PowerShell
+   Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
+   ```
+
+2. The latest version of the Azure PowerShell cmdlets do **NOT** work with this script. You will need to use an earlier version (noted below)
+
+    ```PowerShell
+    Install-Module -Name Az -RequiredVersion 4.2.0 -Force -AllowClobber -SkipPublisherCheck
+    ```
+
+    > Note: If you need to uninstall first: [Uninstall the Azure PowerShell Module](https://docs.microsoft.com/en-us/powershell/azure/uninstall-az-ps)
+
+3. If you installed an update, **close** the PowerShell 7 window, then **re-open** it. This ensures that the latest version of the Az module is used.
+
+4. Execute the following to sign in to the Azure account that has the **Owner** role assignment in your subscription.
+
+    ```PowerShell
+    Connect-AzAccount
+    ```
+
+5. If you have more than one subscription, be sure to select the right one before the next step. Use `Get-AzSubscription` to list them and then use the command below to set the subscription you're using:
+
+    ```powershell
+    Select-AzSubscription -Subscription <The selected Subscription Id>
+    ```
+
+6. Assign the `$sqlpwd` and `$vmpwd` variables in your PowerShell session as **Secure Strings**. Be sure to use a strong password for both. Follow [this link](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/faq#what-are-the-password-requirements-when-creating-a-vm) for Virtual Machine password requirements and [this link](https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-2017#password-complexity) for SQL Server.
+
+    ```powershell
+    $sqlpwd = "ThePasswordYouWantToUseForSQL" | ConvertTo-SecureString -AsPlainText -Force
+    $vmpwd = "ThePasswordYouWantToUseForTheVM" | ConvertTo-SecureString -AsPlainText -Force
+    ``` 
+
+7. If you have not already done so, you will need to download the `LabDeployment` folder from the repository.  You can use the following command to clone the repo to the current directory:
+
+   ```shell
```

Comment:
This is asking the coach (or student?) to clone a repo that is external to WTH.  Are the files being cloned from this 3rd party repo the same as the ones in the /Coach/LabDeployment subfolder?   

If yes, then these instructions should be changed to refer to that location.

If there is any expectation that these files should be provided to the attendees, then the coach should be instructed to download the /Coach/LabDeployment folder and package it into a ZIP file to share with students if needed.